Frontend/ui-driver/src/Flow.purs

```diff
@@ -1693,10 +1697,21 @@ bookingOptionsFlow = do
           shortDescription : item.shortDescription,
           vehicleRating : item.vehicleRating,
           isUsageRestricted : fromMaybe false item.isUsageRestricted,
-          priority : fromMaybe 0 item.priority
+          priority : fromMaybe 0 item.priority,
+          rateCardData : Nothing,
+          perKmRate : Nothing,
+          farePolicyHour : Nothing
         }
       )
 
+rateCardScreenFlow :: FlowBT String Unit
+rateCardScreenFlow = do
+  action <- UI.rateCardScreen
+  case action of
+    TA.REFRESH_RATE_CARD _ -> do
+      modifyScreenState $ RateCardScreenStateType (\rateCardScreen -> rateCardScreen{props{ reRenderSlider = true }})
```

Comment:
@Rohit4997 do we need to exit the screen to rerender slider? Can we remove this if not required.